src/js/angular/controllers.js

```diff
@@ -503,10 +509,10 @@ function mainCtrl($scope, $menuItems, $jwtAuth, $http, toastr, $location, $repos
         ];
         $scope.activePage = 0;
         $(".pages-wrapper .page-slide").css("opacity", 100);
-        const withOfParentElm = $(".pages-wrapper")[0].offsetWidth + 200;
+        const widthOfParentElm = $(".main-container")[0].offsetWidth + 200;
         $timeout(function () {
```

Comment:
This timeout may cause a memory leak. There is a task to fix them, so I suggest to mark the method with a TODO.

Reply:
Since this has nothing to do with security or the fixes I left it as it is. I'm sure it won't be difficult to find all $timeout usage when we get to the task you mentioned.